files/zh-cn/web/api/web_speech_api/using_the_web_speech_api/index.md

```diff
@@ -192,29 +192,25 @@ recognition.onerror = function (event) {
 
 ## Speech synthesis
 
-语音合成 (也被称作是文本转为语音，英语简写是 tts) 包括接收 app 中需要语音合成的文本，再在设备麦克风播放出来这两个过程。
+Speech synthesis（语音合成，也被称作是文本转为语音，英语简写是 TTS）包括接收 app 中需要语音合成的文本，再在设备扬声器或音频输出连接中播放出来这两个过程。
 
 Web Speech API 对此有一个主要控制接口——[`SpeechSynthesis`](/zh-CN/docs/Web/API/SpeechSynthesis) ，外加一些处理如何表示要被合成的文本 (也被称为 utterances)，用什么声音来播出 utterances 等工作的相关接口。同样的，许多操作系统都有自己的某种语音合成系统，在这个任务中我们调用可用的 API 来使用语音合成系统。
 
 ### Demo
 
-为了展示 Web 语音合成的简单使用，我们提供了一个例子——[Speak easy synthesis](https://github.com/mdn/dom-examples/tree/main/web-speech-api/speak-easy-synthesis) 。例子是一套表单控件，包括输入需要被合成的文本，设置音调、语速和发出文本时需要的语音。在输入文本之后，按下`Enter`/`Return`键使它播放。
+为了展示 Web 语音合成的简单使用，我们提供了一个例子 —— [Speak easy synthesis](https://github.com/mdn/dom-examples/tree/main/web-speech-api/speak-easy-synthesis) 。例子是一套表单控件，包括输入需要被合成的文本，设置音调、语速和说出文本时需要的语音。在输入文本之后，按下 <kbd>Enter</kbd>/<kbd>Return</kbd> 键使它播放。
 
 ![UI of an app called speak easy synthesis. It has an input field in which to input text to be synthesised, slider controls to change the rate and pitch of the speech, and a drop down menu to choose between different voices.](speak-easy-synthesis.png)
 
 想跑这个例子，你可以 git clone Github 仓库中的部分 (或者[直接下载](https://github.com/mdn/dom-examples/archive/refs/heads/main.zip))，在桌面版支持的浏览器打开 index.html 文件，或者在移动端浏览器直接导向 [live demo URL](https://mdn.github.io/dom-examples/web-speech-api/speak-easy-synthesis/) ，像 Chrome 和 Firefox OS。
 
-### 浏览器支持
+### 浏览器兼容性
 
-Web Speech API 语音合成部分在各浏览器中还是在发展，还不成熟，现在有以下几个限制点：
-
-- Firefox 桌面版和移动版在 Gecko 42+(Windows)/44+ 支持，但是没有前缀，可以通过将`media.webspeech.synth.enabled`标志在`about:config`中转为`true`来启用。
-- Firefox OS 2.5+ 支持，但是默认的，不需要任何权限。
-- Chrome 桌面版和安卓版自 33 版以来都支持，但是没有前缀
+[api.SpeechSynthesis](/zh-CN/docs/Web/API/Web_Speech_API#api.speechsynthesis)
```

Comment:
I could not see this section in en-US, please remove it instead of updating it.

Reply:
Agreed